### DISCUSSION TITLE
Why is the maximum number of client connections 256?

### DISCUSSION THREAD

**Discussion by sqlnew** [jchristn/WatsonWebsocket#104] [Q&A]
Use the client to connect to the websocket server in the browser. When there are more than 256 connections, you can no longer connect. Does the server limit the number of client connections to 256? Can this limit be adjusted? Because 256 connections are too few for our business. Look forward to your reply Thank you!

**Comment by jchristn** [comment]
There is no such limit imposed in the code.  If there is some limit, it would have to be in the underlying libraries provided by Microsoft.  Cheers, moving to discussion.
